wcfsetup/install/files/lib/system/html/output/HtmlOutputProcessor.class.php

```diff
@@ -47,6 +47,12 @@ class HtmlOutputProcessor extends AbstractHtmlProcessor {
 	 */
 	protected $outputType = 'text/html';
 	
+	/**
+	 * enables rel=ugc for external links
+	 * @var bool
+	 */
+	protected $ugc = true;
```

Comment:
While I'm not overly happy with this being a class property (it can lead to subtle bugs if the HtmlOutputProcessor) is reused there is precedent in the `removeLinks` property. That one is `public` however, without any getters / setters. Should those be added for `removeLinks` for consistency?